src/material/input/input.md

```diff
@@ -10,7 +10,18 @@ on elements inside `<mat-form-field>` as well. This includes Angular directives
 and `formControl`.
 
 The only limitation is that the `type` attribute can only be one of the values supported by
-`matNativeControl`.
+`matNativeControl`. 
+
+### `matNativeControl` and `matInput` explained
```

Comment:
I'm not sure if this is entirely correct. `matInput` and `matNativeControl` are the same directive, we just have two selectors for historical reasons. It also doesn't necessarily "enhance" the input, but rather it allows `mat-form-field` to find it.

Reply:
Would it be worth adjusting my content to more align with that explanation or rather closing the pr? (is this useful info im adding?) :)